R/plumber.R

```diff
@@ -118,7 +118,7 @@ Plumber <- R6Class(
           private$disable_run <- FALSE
         }, add = TRUE)
 
-        for (i in 1:length(private$parsed)){
+        for (i in seq_len(length(private$parsed))) {
```

Comment:
While plumbing an empty file or a file without any block expression, this would create an error since the loop would go from 1:0 instead of not looping.